lib/consumer.ex

```diff
@@ -100,6 +126,7 @@ defmodule GenRMQ.Consumer do
   @callback init() :: [
               connection: keyword | {String.t(), String.t()} | :undefined | keyword,
               queue: String.t(),
+              queue_options: list,
```

Comment:
Let's change this to `keyword` (and the same on line 140).

Reply:
Changed